gatsby-node.js

```diff
@@ -11,6 +11,21 @@ exports.onPreBootstrap = async () => {
   }
 }
 
+exports.onCreatePage = ({ page, actions }) => {
+  const { deletePage, createPage } = actions
+
+  if (page.path === '/dev-404-page/') {
```

Comment:
Workaround. By default Gatsby has a "development 404" page and you have to click there on the button "Show original 404 page" to see the template. It's a kind of "development" mode for Gatsby - that the app displays development 404 page first instead of custom 404 page and Maria requires the custom 404 page first.